libraries/AP_HAL_ChibiOS/hwdef/BlitzWingH743/README.md

```diff
@@ -0,0 +1,102 @@
+# iFlight BLITZ Wing H743 Flight Controller
+
+The BLITZ Wing H743 is a flight controller produced by [iFlight](https://shop.iflight.com/electronics-cat27/BLITZ-Wing-H743-Flight-Controller-Pro2174).
+
+## Features
+
+ - MCU - STM32H743 32-bit processor running at 480 MHz
+ - Gyro: ICM42688
+ - SDCard for logging
+ - 5V 3A BEC for Flight Controller
+ - 9V 3A BEC for VTX
+ - 5-8.4V 6A BEC for Servo
+ - Barometer: DPS310
+ - OSD: AT7456E
+ - 7x UARTs
+ - 13x PWM Outputs (12 Motor Output, 1 LED)
+ - Battery input voltage: 2S-8S
+ - 2x I2C for external compass, airspeed, etc.
+ - CAN port
+
+## Pinout
+
+![BLITZ Wing H743 Board](blitz_h7_wing_top.PNG "BLITZ Wing H743 Top")
+![BLITZ Wing H743 Board](blitz_h7_wing_middle.PNG "BLITZ Wing H743 Middle")
+![BLITZ Wing H743 Board](blitz_h7_wing_bottom.PNG "BLITZ Wing H743 Bottom")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+|Name|Pin|Function|
+|:-|:-|:-|
+|SERIAL0|COMPUTER|USB|
+|SERIAL1|RX1/TX1|UART1 (DJI connector, DMA-enabled)|
+|SERIAL2|TX2/RX2|UART2 (RX, DMA-enabled)|
+|SERIAL3|TX3/RX3|UART3 (DMA-enabled)|
```

Comment:
this uart is not shown to be pinned out on their board...I suspect the silkscreen is wrong showing T2 instead of T3 on the dupont row on the side....

Reply:
Mine has T3/R3 and T2/R2 on the top side

Reply:
okay, did not see solder pads for it